### PR TITLE
Add template for content proposals for the KubeVirt summit

### PR DESCRIPTION
**Pull request by codificat**
An initial basic template to submit content proposals for the upcoming KubeVirt online event

### CODE REVIEW

**Comment by codificat** [comment]
@davidvossel FYI. Should we add or remove anything?

File: events/2021-kubevirt-summit/proposals/proposal-template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Abstract
+
+Explain what the session will be about, what will be covered, what will be the
+benefits of attending it / what can attendees expect to get from it).
+
+# Presenters
+
+- Name, title, company, contact details (email, twitter, github - pick any/all)
+
+[ ] The presenters agree to abide by the KubeVirt Code of Conduct
```
Comment (davidvossel) [inline events/2021-kubevirt-summit/proposals/proposal-template.md:14]: maybe link? https://github.com/kubevirt/community/blob/master/code-of-conduct.md

**Review by davidvossel — COMMENTED** [review]
Looks great to me, maybe link to code of conduct since we reference it here.

File: events/2021-kubevirt-summit/proposals/proposal-template.md (hunk 1 above)
Comment (codificat) [reply events/2021-kubevirt-summit/proposals/proposal-template.md]: For events, the CNCF uses the Linux Foundation code of conduct for events, which contains specific information around incidents:
https://events.linuxfoundation.org/about/code-of-conduct/
I believe we should link here instead

Comment (davidvossel) [reply events/2021-kubevirt-summit/proposals/proposal-template.md]: yup, looks good to me

**Review by davidvossel — APPROVED** [review]
/lgtm
/approve

**Pull request merged**